projects/rero/ng-core/src/lib/record/search-autocomplete/search-autocomplete.component.ts

```diff
@@ -58,7 +58,7 @@ export interface IAutoComplete {
       [minLength]="minLength()"
       [placeholder]="placeholder()"
       [ngModel]="value()"
-      (ngModelChange)="currentValue = $event"
+      (ngModelChange)="currentValue = sanitize($event)"
```

Comment:
You can create a pipe.

Reply:
the pipe in a function return for transformation does not work